src/tests/ftest/util/data_mover_test_base.py

```diff
@@ -373,6 +395,175 @@ def gen_uuid(self):
             new_uuid = str(uuid.uuid4())
         return new_uuid
 
+    def parse_create_cont_uuid(self, output):
+        """Parse a uuid from some output.
+
+        Format:
+            Successfully created container (.*-.*-.*-.*-.*)
+
+        Args:
+            output (str): The string to parse for the uuid.
+
+        Returns:
+            str: The parsed uuid.
+
+        """
+        uuid_search = re.search(
+            r"Successfully created container (.*-.*-.*-.*-.*)",
+            output)
+        if not uuid_search:
+            self.fail("Failed to parse container uuid")
+        return uuid_search.group(1)
+
+    def dataset_gen(self, cont, num_objs, num_dkeys, num_akeys_single,
+                    num_akeys_array, akey_sizes, akey_extents):
+        """Generate a dataset with some number of objects, dkeys, and akeys,
```

Comment:
You might want to check out this description style examples that we're trying to follow.
https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

The description summary should fit in the first line, although sometimes I find it hard to fit in a single line. The detailed description paragraph has no indentation inside the comment block.

Reply:
Ah, thanks for that! I'll start following that going forward.